src/pages/postgraphile/smart-comments.md

```diff
@@ -224,9 +224,17 @@ X | **`execute`**	| -	| -	| function not present
 
 </div>
 
-
 > **Warning:** This functionality is not intended for implementing permissions, it's for removing things from your API that you don't need. You should back these up with database permissions if needed.
 
+#### Unique-key record finders
+To exclude Unique-key record finders (_e.g._ `photoByUrl` or `personById`) from your GraphQL schema, add an `@omit` comment on the constraint
+
+```sql
+comment on constraint person_pkey on person is 
+  E'@omit';
+```
+>**Note:** Only `@omit` alone does the desired job. Listing any operation(s) (`create`, `filter`, `many`...) will yield no results.
+
```

Comment:
That shouldn’t be the case... could you file an issue about it and I’ll look into it. “@omit read,...” should definitely work. “@omit filter” should remove the “filter” arg from the field. “@omit create” doesn’t make sense on a unique relation so shouldn’t make any difference, but “@omit update” should prevebt the “updatePhotoByUrl” mutation being generated

Reply:
Cool, I'll look into it!

Reply:
Yeah, turns out that since I already omitted update,delete from the tables overall, the comment didn't do anything, my bad! Hey... what should `@omit read` do?

Reply:
You're right, my mistake. I didn't realize I had the operations omitted on the table. 
Delete and update work as expected, I don't know how order/filter/many/all should work though. 
Read is broken for me, see [#813](https://github.com/graphile/postgraphile/issues/813)